bedrock/mozorg/templates/mozorg/internet-health.html

```diff
@@ -81,6 +81,17 @@
     <section id="innovation" class="even">
       <div class="content">
         <div class="inner-container">
+        {% if l10n_has_tag('internet_health_digital_inclusion') %}
```

Comment:
Yes, apologies, this isn't new text after all. It was fixed in ee5105c25b063506875a086a49203deddb72d411.